library/src/interactor_impl.cxx

```diff
@@ -693,6 +693,30 @@ interactor& interactor_impl::initCommands()
         this->Internals->AnimationManager->GetAnimationIndex();
     });
 
+  this->addCommand("cycle_anti_aliasing",
+    [&](const std::vector<std::string>&)
+    {
+      bool& enabled = this->Internals->Options.render.effect.antialiasing.enable;
+      std::string& mode = this->Internals->Options.render.effect.antialiasing.mode;
+      if (!enabled)
+      {
+        enabled = true;
+        mode = "fxaa";
+      }
+      else
+      {
+        if (mode == "fxaa")
+        {
+          mode = "ssaa";
+        }
+        else
+        {
+          enabled = false;
+        }
+      }
```

Comment:
I wonder if there is a better way to implement a cycle like this, but cant think of anything not overengineered

Reply:
we should probably implement something for generic enums at some point with a unique `cycle` command, but it's out of the scope of this PR.